### PR TITLE
channel list change log update

### PR DESCRIPTION
**Pull request by DanC5**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.3.0] 2020-09-01
```
Comment (vishalnarkhede) [inline CHANGELOG.md:3]:
```suggestion
## [1.3.0] 2020-09-01 (UNPUBLISHED)
```

Comment (vishalnarkhede) [reply CHANGELOG.md:3]: Also all the rest of the changes please e.g., context related changes that were made in this release.

Comment (DanC5) [reply CHANGELOG.md:3]: oh ok, I must have misunderstood, I thought you were just referencing the ChannelList stuff, not all the Channel context stuff, but can add no problem!

File: CHANGELOG.md [hunk 2]
```diff
@@ -415,7 +419,7 @@ From now on, autocomplete input (specifically mentions autocomplete) will be api
 
 ## [0.7.0] 2020-03-17
 
-- Introducing internationalisation (i18n) support for the sdk https://github.com/GetStream/stream-chat-react-native/blob/master/src/components/docs/Streami18n.md
+- Introducing internationalization (i18n) support for the sdk https://github.com/GetStream/stream-chat-react-native/blob/master/src/components/docs/Streami18n.md
```
Comment (virdesai) [inline CHANGELOG.md:422]: lol American version of spelling

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.3.0] 2020-09-01 (UNPUBLISHED)
+
+- We converted the ChannelList component from a class to a function. We also abstracted the event listener logic into custom hooks. On upgrading to this release, ensure events and any custom handling functions (ex: `onAddedToChannel` or `onMessageNew`) are properly processed and update the UI as expected.
+
```
Comment (vishalnarkhede) [inline CHANGELOG.md:6]: Lets add a detailed explanation - similar to https://github.com/GetStream/stream-chat-react/blob/master/CHANGELOG.md#110-2020-06-08

Also, please add the breaking changes regarding context:

1. Which new contexts are introduced
2. Which context values are moved from where to where

**Pull request merged**